packages/backend/server/schema.prisma

```diff
@@ -443,6 +447,25 @@ model AiPrompt {
   @@map("ai_prompts")
 }
 
+model AiSession {
+  id          String   @id @default(uuid()) @db.VarChar
+  userId      String   @map("user_id") @db.VarChar
+  workspaceId String   @map("workspace_id") @db.VarChar
+  docId       String   @map("doc_id") @db.VarChar
+  promptName  String   @map("prompt_name") @db.VarChar
+  action      Boolean  @db.Boolean
+  model       String   @db.VarChar
+  messages    Json     @db.Json
```

Comment:
should we put all messages in one session record, which would be too large for a long context services.

Reply:
now split messages as independent table